interfaces/matlab/sim/generate_c_code_explicit_ode.m

```diff
@@ -0,0 +1,81 @@
+%   This file is part of acados.
```

Comment:
Do I get it right that this replaces all the custom scripts we had to generate every model?? Finally!! 👍 Maybe a short wiki entry with an example on how to use it?

Reply:
Could also be extended to automatically generate a header file and the lines to use `external_function_casadi_create`, but might be complementary work to the Matlab interface...
@dkouzoup I dont have rights for the wiki..

Reply:
Sorry, I just invited you. Actually at the moment the code generator is in C++ and we are thinking for various reasons to move it to matlab. Therefore, if you implement it it will for sure be very useful!